PowerEditor/installer/nativeLang/bulgarian.xml

```diff
@@ -1086,7 +1086,7 @@
 
 Продължаване?"/>
 			<CannotMoveDoc title="Преместване в нов прозорец на Notepad++" message="Документа е променен, запишете го и опитайте отново."/>
-			<DocReloadWarning title="Презареждане" message="Сигурни ли сте, че искате да презаредите текущя документ и да загубите промените, направени с Notepad++?"/>
+			<DocReloadWarning title="Презареждане" message="Сигурни ли сте, че искате да презаредите текущия документ и да загубите промените, направени с Notepad++?"/>
```

Comment:
Correct

Reply:
So @rddim could you make this correction in your next PR ?

Reply:
Done